### ISSUE TITLE
How to get metrics.html without timestamp?

### ISSUE THREAD

**Issue by Tset-Noitamotua** [adiralashiva8/robotframework-metrics#77]
Would be great to have the timestamp in generated result file optionally.

**Comment by adiralashiva8** [comment]
This feature is addressed as part of https://github.com/adiralashiva8/robotframework-metrics/pull/73 pull request and this change will be available in next release 3.1.3, either user can install these changes by using following command
```
pip install git+https://github.com/adiralashiva8/robotframework-metrics
```

command to modify report name:
```
robotmetrics -M myreportname.html [otherOptions:]
```

**Comment by Tset-Noitamotua** [comment]
This one is solved. Thank You!
